src/components/ReportModal/ReportSection/ReportSection.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC, ReactNode } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { IconDefinition } from '@fortawesome/fontawesome-common-types';
+import styles from './ReportSection.module.scss';
+
+interface ReportSectionProps {
+  icon: IconDefinition;
+  fields: { name: ReactNode; value: ReactNode }[];
+}
+
+const ReportSection: FC<ReportSectionProps> = ({ icon, fields }) => {
+  return (
+    <div className={styles.sectionContainer}>
+      <div className={styles.sectionIconContainer}>
+        <FontAwesomeIcon icon={icon} className={styles.icon} />
+      </div>
+      <div className={styles.fieldContainer}>
+        {fields.map(field => (
+          <ReportField sectionTitle={field.name} value={field.value} />
```

Comment:
This should probably include a `key`.
```suggestion
          <ReportField sectionTitle={field.name} value={field.value} key={field.name} />
```

Reply:
Good call I missed that error somehow.

It was a little trickier than just using `field.name` since it is typed as a `ReactNode` and `key` requires a `string | number | undefined`.  I think it's desirable to leave name as a `ReactNode` so that a complex HTML element can be used as a name (maybe something in bold or with an icon).

instead of using the index as a key ([generally bad](https://reactjs.org/docs/lists-and-keys.html#keys)) or manually specifying the keys I opted to add and use `shortid` which is the preferred option in the article linked in the react docs.

I also did add filtering for any section that has a falsey name while working on this.